src/main/java/step2/domain/WinningInfos.java

```diff
@@ -0,0 +1,38 @@
+package step2.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinningInfos {
+
+    private List<WinningInfo> winningInfoList = new ArrayList<>();
+    private static final int PRICE_PER_LOTTO = LottoMachine.PRICE_PER_LOTTO;
```

Comment:
관례상 상수가 제일 먼저 위치해있고
그다음 멤버 변수, 생성자 순입니다. 협업 시에는 관례를 따라주는 것이 좋을 것 같습니다.

```suggestion
    private static final int PRICE_PER_LOTTO = LottoMachine.PRICE_PER_LOTTO;
    
    private List<WinningInfo> winningInfoList = new ArrayList<>();
```